.github/workflows/push.yml

```
@@ -720,12 +720,72 @@ jobs:
+        run: '[[ "${{ steps.infra-node-modules.outputs.success }}" != "false" ]] || exit 1'
+      - name: Install infra modules
+        if: steps.infra-node-modules.outputs.cache-hit != 'true'
+        working-directory: monorepo/infra
+        run: |
+          CI=true yarn install --immutable
+      - name: Generate islandis feature-deployment charts for ArgoCD
+        env:
+          FEATURE_NAME: ${{ needs.pre-checks.outputs.FEATURE_NAME }}
+          IMAGES: ${{ needs.prepare.outputs.IMAGES }}
+          DOCKER_TAG: ${{ needs.prepare.outputs.DOCKER_TAG }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:68: Double quote to prevent globbing and word splitting

(shellcheck)